src/Core/src/Platform/iOS/MauiView.cs

```diff
@@ -134,6 +134,7 @@ public override void LayoutSubviews()
 			}
 
 			CrossPlatformArrange(bounds);
+			OnLayoutChanged();
```

Comment:
Maybe this should be renamed to match the method. Also, no need to have the method, just invoke the event directly. Maybe use a name of `ArrangingSubviews` or even just `OnLayoutSubviews` since it is internal. I see `MovedToWindow` got away with the same name because it was an explicit interface member.

Reply:
I changed to invoke the event directly